docker-compose.yaml

```diff
@@ -212,7 +212,8 @@ services:
       - TURBO_UPLOAD_SERVICE_URL=${TURBO_UPLOAD_SERVICE_URL:-}
       - RUN_OBSERVER=${RUN_OBSERVER:-true}
       - MIN_RELEASE_NUMBER=${MIN_RELEASE_NUMBER:-0}
-      - AR_IO_NODE_RELEASE=${AR_IO_NODE_RELEASE:-29-pre}
+      - AR_IO_NODE_RELEASE=${AR_IO_NODE_RELEASE:-28-pre}
```

Comment:
This shouldn't be changing.

Reply:
looks like i overwrote this on the rebase with develop. updating